lib/renderer/inspector.js

```diff
@@ -4,10 +4,19 @@ window.onload = function () {
   // Use menu API to show context menu.
   window.InspectorFrontendHost.showContextMenuAtPoint = createMenu
 
+  // correct for Chromium returning undefined for filesystem
+  window.Persistence.FileSystemWorkspaceBinding.completeURL = completeURL
+
   // Use dialog API to override file chooser dialog.
   window.UI.createFileSelectorElement = createFileSelectorElement
 }
 
+// Extra / is needed as a result of MacOS requiring absolute paths
+function completeURL (project, path) {
+  project = 'file:///'
```

Comment:
Wouldn't this be more robust?
```
url.format({
    protocol: 'file',
    slashes: true,
    pathname: `${project}${path}`
})
```

Reply:
@miniak we need the weird `///` (3 slashes) and we don't want to double escape the path (`url.format` would escape again)

Reply:
got it